src/App/Pages/Send/SendAddEditPageViewModel.cs

```diff
@@ -231,6 +233,8 @@ public bool SendOptionsPolicyInEffect
         public bool ShowDeletionCustomPickers => EditMode || DeletionDateTypeSelectedIndex == 6;
         public bool ShowExpirationCustomPickers => EditMode || ExpirationDateTypeSelectedIndex == 7;
         public string ShowPasswordIcon => ShowPassword ? BitwardenIcons.EyeSlash : BitwardenIcons.Eye;
+        public string FileTypeAccessibilityLabel => $"{_i18nService.T("File")} {_i18nService.T(IsFile ? "TypeIsSelected" : "TypeIsNotSelected")}";
+        public string TextTypeAccessibilityLabel => $"{_i18nService.T("Text")} {_i18nService.T(IsText ? "TypeIsSelected" : "TypeIsNotSelected")}";
```

Comment:
Composition should be avoided in strings translations as this because it's just the way it's written in english but in another language it could vary like in Spanish: "Archivo tipo es seleccionado" makes no sense.
So, in this case you should have "FileTypeIsSelected", "TextTypeIsSelected", "FileTypeIsNotSelected" and "TextTypeIsNotSelected" as keys so they can be well translated to any language.

Reply:
Also `_is18nService` is not necessary, you can directly call `AppResources.File`

Reply:
Totally agree.